src/magnet_coils.py

```diff
@@ -0,0 +1,582 @@
+import log
+import cubit
+import numpy as np
+from sklearn.preprocessing import normalize
+from pathlib import Path
+import subprocess
+import yaml
+import argparse
+
+m2cm = 100
+
+def unit_vector(vec):
+    """Normalizes given vector.
+
+    Arguments:
+        vec (array of [float, float, float]): input vector to be normalized.
+
+    Returns:
+        vec (array of [float, float, float]): normalized input vector.
+    """
+    vec = normalize(vec.reshape(-1, 1), axis=0).ravel()
```

Comment:
I wonder if we can do this without an extra dependency on `sklearn` for this?  We put something together based on only `numpy` in the slice that @connoramoreno is working on.

Reply:
yeah it can be done with numpy